fairlearn/reductions/_moments/regression_moment.py

```diff
@@ -0,0 +1,134 @@
+from .utility_parity import UtilityParity
+from .moment import Moment,_LABEL,_GROUP_ID
+from .weighted_error_rate import WeightedErrorRate
+from .bounded_group_loss import SquareLoss
+import pandas as pd
+import numpy as np
+from itertools import repeat
+
+class CDF_DemographicParity(Moment):
+    short_name = "CDF_DemographicParity"
+
+    def __init__(self,loss,y_range,difference_bound=None,grids=[],grid_num=41):
+        super().__init__()
+        self.utility_parity = UtilityParity(difference_bound=difference_bound)
+        self.loss = loss
+        self.objective = WeightedErrorRate(self.loss) #loss: SquareLoss, AbsoluteLoss, implemented inside the bounded_group_loss.py
```

Comment:
sounds like you'd want to move these mentioned losses out of bounded_group_loss.py, any reason why not?

Reply:
I think it is plausible to move them out, but I am afraid that these changes will affect other components in the fairlearn library and I do not have the right to make this change.

Reply:
What do you mean? Moving small parts around is fairly straightforward. You just need to update the references to it.